app/utils/chainUtils.ts

```diff
@@ -147,7 +133,7 @@ export const evmMethods = [
 ]
 
 export const isEvmChain = (chain: Blockchain | null): boolean =>
-  !!chain && evmChains.includes(chain.blockchain)
+  !!chain && evmChains.includes(chain.id)
```

Comment:
#PUC why this should be `.id` instead of `.blochain`

Reply:
Done.